shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -421,6 +423,120 @@ public Intent build(@NonNull Context context) {
     }
   }
 
+  /**
+   * Creates a {@link NewEngineInGroupIntentBuilder}, which can be used to configure an {@link
+   * Intent} to launch a {@code FlutterActivity} that internally uses an existing {@link
+   * io.flutter.embedding.engine.FlutterEngineGroup} that is cached in {@link
+   * io.flutter.embedding.engine.FlutterEngineGroupCache}. and creates a new {@link
```

Comment:
to launch a FlutterActivity by internally creating a FlutterEngine from an existing FlutterEngineGroup cached in a specified FlutterEngineGroupCache